tests/test_amend.py

```diff
@@ -162,6 +162,7 @@ def test_supplier_already_present(self) -> None:
     def test_publisher_already_present(self) -> None:
         component = {"author": "x", "publisher": "y"}
         expected = copy.deepcopy(component)
+        expected["supplier"] = "y"
```

Comment:
This should not be the case. The supplier should not change when already existing.

Reply:
There is no supplier, only publisher and author